Python/_warnings.c

```diff
@@ -1566,7 +1568,9 @@ PyErr_WarnEx_WithFix(PyObject *category, const char *msg_txt, const char *fix_tx
     int ret;
     PyObject *message = PyUnicode_FromString(msg_txt);
     PyObject *fix = PyUnicode_FromString(fix_txt);
-    if (message == NULL || fix == NULL)
+    if (message == NULL)
```

Comment:
*Technically* `message` could be `NULL` without `fix` being `NULL`. It would be safer to do something like:

```
PyObject *message = PyUnicode_FromString(msg_txt);
if (message == NULL) return -1;
PyObject *fix = PyUnicode_FromString(fix_txt);
if (fix == NULL) return -1;
```

Reply:
I have made these changes